res/css/views/rooms/_EventTile.scss

```diff
@@ -653,3 +653,17 @@ div.mx_EventTile_notSent.mx_EventTile_redacted .mx_UnknownBody {
         }
     }
 }
+
+.mx_EventTile_tileError {
+    color: red;
+
+    .mx_EventTile_line span {
+        padding: 4px 8px;
+        border-radius: 11px;
+        box-shadow: 0px 0px 3px red inset;
+    }
+
+    a {
+        margin-left: 1em;
+    }
+}
```

Comment:
newline

Reply:
Fixed, and fixed the underlying issue too

<img width="555" alt="Screenshot 2020-02-25 at 10 21 01" src="https://user-images.githubusercontent.com/3935731/75238223-8aa6af00-57b8-11ea-8b38-7e01219e8f9b.png">